src/services/configService.js

```diff
@@ -69,4 +63,22 @@ export class ConfigService {
     return HttpClient.put(`${ENDPOINT}solution-settings/logo`, logo, options)
       .map(prepareLogoResponse);
   }
+
+  /* get solution settings */
+  static getSolutionSettings() {
+    return HttpClient.get(`${ENDPOINT}solution-settings/theme`)
+      .map(toSolutionSettingThemeModel)
+      /* When the application loads for the first time, there won't be "solution-settings"
+       in the storage. In that case, service will throw a 404 not found error.
+       But we need to ignore that and stick with the static values defined in UI state */
```

Comment:
In this situation, create a const for the default values (see authDisabledUser in [models/authModels.js](models/authModels.js) as an example.  Then return that default from this service call instead of the exception.

Also, be sure to check the value you are currently returning.  Now that its the solution settings and not a single value, I don't think "true" is correct anymore.